features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -525,6 +525,13 @@
 				<AttributeID>failedLoginAttempts</AttributeID>
 				<Description>Failed Login Attempts</Description>
 			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/identity/failedLoginAttemptsBeforeSuccess</ClaimURI>
```

Comment:
We need to come up with a better name for this. Shall we discuss this in slack?

Reply:
Did we come up with a suitable name? Any discussion happened?